tsconfig.json

```diff
@@ -4,10 +4,12 @@
     "module": "commonjs",
     "sourceMap": true,
     "outDir": "dist",
-    "strict": false,
-    "allowJs": true,
+    "strict": true,
+    "allowJs": false,
```

Comment:
I didn't convert it because after talking with some people in the mvp channel we were uncertain if we would still use this file/feature. Since the file isn't imported anywhere it doesn't affect the build